packages/meteor-otel/tests/tracing_tests.js

```
@@ -0,0 +1,575 @@
+  withSpanSync,
+  createSpanBuilder,
+  extractTraceContext,
+  injectTraceContext,
+  getTraceContext,
+  createContextFromTrace,
+  runWithTraceContext,
+  createLinkedSpan,
+  getTracer,
+  trace,
+  context,
+} from 'meteor/meteor-otel';
```

ESLint

[error] `meteor/meteor-otel` import should occur before import of `meteor/tinytest`

(import/order)